.github/workflows/ci.yml

```diff
@@ -131,13 +131,13 @@ jobs:
       matrix:
         include:
           - type: "general-superstaq"
-            dependencies: "./general-superstaq"
+            dependencies: "-e ./general-superstaq[dev]"
           - type: "qiskit-superstaq"
-            dependencies: "./general-superstaq ./qiskit-superstaq[dev]"
+            dependencies: "-e ./general-superstaq -e ./qiskit-superstaq[dev]"
           - type: "cirq-superstaq"
-            dependencies: "./general-superstaq ./qiskit-superstaq ./cirq-superstaq[dev]"
+            dependencies: "-e ./general-superstaq -e ./qiskit-superstaq -e ./cirq-superstaq[dev]"
           - type: "supermarq-benchmarks"
-            dependencies: "./general-superstaq ./qiskit-superstaq ./cirq-superstaq ./supermarq-benchmarks[dev]"
+            dependencies: "-e ./general-superstaq -e ./qiskit-superstaq -e ./cirq-superstaq -e ./supermarq-benchmarks[dev]"
```

Comment:
Just for understanding, but I recall we had to [get rid of](https://github.com/Infleqtion/client-superstaq/pull/940/files) the editable installs. What was the key issue previously?

Reply:
hmm not sure, but tbh not sure why i made this change either so reverted

Reply:
My best guess if I recall correctly was that this breaks during client version bump because the new version isn't deployed yet or something